docs/helm-broker/03-01-create-bundles.md

```diff
@@ -112,6 +112,8 @@ The `meta.yaml` file contains the specification of the ClusterDocsTopic. The exa
 
 >**NOTE:** Currently you can provide only one entry in the `docs` array.
 
+See [this](https://github.com/kyma-project/bundles/tree/master/bundles/testing-0.0.1/docs) example of the `docs` directory with documentation for the testing bundle.
```

Comment:
I like this reference but above there is no mention about docs directory containing markdown and assets.

Reply:
I added an explanation regarding the markdown documents and assets directory. Also mentioned  AsyncAPI, OData, and OpenAPI specification files.